### ISSUE TITLE
Direct Input of Logitech G29 throttle

### ISSUE THREAD

**Issue by LukasMaier1604** [csutorasa/XOutput#178]
First of all i must say this is a great application. My Problem currently is, that i tried to convert the direct Input of me Logitech G29 Steering wheel to XOutput. Unfortunately the app doesn´t recognize the throttle as a direct input. Every other Button/ Axis works fine. Could you maybe resolve this.

**Comment by Spoonos** [comment]
I'm having a similar issue where XOutput isn't recognizing the brake on my Saitek R440 wheel

**Comment by csutorasa** [comment]
I know that there are issues with a few wheels where one of the pedals go wrong. No idea so far how to fix it as I have none of these devices. I will keep trying to find out a way to fix this with other ways.

**Comment by efortes** [comment]
Yes I have the same issue. I wanted to use this for google stadia. Every other button works.
Will not be fixed right?

**Comment by FahrtrichtungLinks** [comment]
csutorasa, please tell me you're gonna fix this issue. :)

I just ordered the G29 and plan to use it a lot with XOutput.

If we can help somehow or donate something, let us know.

Thanks for the software. :)

**Comment by 2nt0** [comment]
hey all with this issue (and csutorasa if you're reading this) - i have found that v3.18 works perfectly with g29 throttle pedal, and only v3.19 are affected (i have the issue where the axis is showing up but permanently reading 0, but _only in v3.19+_) as a temporary workaround, roll back to an older non-affected version. csutorasa - if you're still having trouble fixing this (i have not checked v4 developer builds so you may well have solved this issue), it is something that was changed in v3.19. have also copied this message into the other g29 throttle thread.

**Comment by EpochalEngineer** [comment]
I confirmed it works in 3.18 and does not in 3.19 or 3.32 on my G29s
